ci/datamgr.py

```diff
@@ -136,6 +142,15 @@ def download(base_url, directory, name):
     if path.suffix in ('.tar', '.gz', '.bz2', '.xz'):
         with tarfile.open(str(path), mode='r|gz') as f:
             f.extractall(path=str(directory))
+    # get data for geospatial support tests
+    geo_url = (
+          'https://raw.githubusercontent.com/Quansight/ibis/'
+          'geospatial_functions/ci/ibis-testing-data/geo.csv'
```

Comment:
Please create a PR to https://github.com/ibis-project/testing-data with the geo.csv added and change the download script to fetch `https://api.github.com/repos/ibis-project/testing-data/tarball/master`

Reply:
@kszucs that sounds pretty good .. I am going to do that now :+1:

Reply:
PR opened https://github.com/ibis-project/testing-data/pull/1

Reply:
I will now change `download` function

Reply:
Merged :)